### ISSUE TITLE
Doesn't work with ACF

### ISSUE THREAD

**Issue by navitronic** [crowdfavorite/wp-admin-column-view#3]
The plugin seems to break Advanced custom fields when both are enabled.

The Custom Fields admin menu item contains a broken link when this plugin is enabled.

**Comment by stompweb** [comment]
When/if #7 is merged you'll be able to do the following:

``` php
function remove_post_types_admin_column_view($post_types) {

    unset($post_types['acf']);

    return $post_types;
}

add_filter('cf_admin_column_post_types', 'remove_post_types_admin_column_view');
```

**Comment by alexkingorg** [comment]
Fixed in #9 - assuming the filter code above is used. Willing to review other ACF compatibility issues via future pull request.
